script/openqa-websockets

```diff
@@ -15,6 +15,7 @@
 # You should have received a copy of the GNU General Public License along
 # with this program; if not, see <http://www.gnu.org/licenses/>.
 
+use warnings;
 use Mojo::Base -strict;
```

Comment:
This should indeed use `strict` and `warnings` manually, or keep using `Mojo::Base -strict` to do both.